.gitignore

```diff
@@ -74,14 +74,14 @@ static/images/marketplace
 # security rules
 content/en/security/default_rules/*.md
 !content/en/security/default_rules/_index.md
-content/en/security/cloud_workload_security/agent_expressions.md
-content/en/security/cloud_workload_security/backend.md
+content/en/security/threats/agent_expressions.md
+content/en/security/threats/backend.md
 content/en/security_platform/
 
 # cspm cloud resource schemas
-content/en/security/cspm/custom_rules/*.md
-!content/en/security/cspm/custom_rules/_index.md
-!content/en/security/cspm/custom_rules/schema.md
+content/en/security/misconfigurations/custom_rules/*.md
+!content/en/security/misconfigurations/custom_rules/_index.md
+!content/en/security/misconfigurationspm/custom_rules/schema.md
```

Comment:
Should I also keep the old paths listed here (temporarily)? My concern is that others on the team may inadvertently commit these files to the repo (in the old directories) if they haven't pulled latest from master.

Reply:
Yes, it can't hurt to keep both paths as a safety to make sure someone with an old branch doesn't accidentally commit them